FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/V2ListItemActivity.kt

```diff
@@ -275,10 +281,49 @@ private fun CreateListActivityUI(context: Context) {
                 border = BorderType.Bottom
             )
             ListItem.SectionDescription(description = "Centered action text only supports primary text and ignores any given trailing or leading accessory Contents")
+            GroupedList()
         }
     }
 }
 
+@Composable
+private fun GroupedList() {
+    ListItem.Header("Grouped List")
+    ListItem.SectionDescription(description = "Grouped List", modifier = Modifier.height(25.dp))
+    Column(modifier = Modifier.padding(horizontal = 10.dp, vertical = 10.dp).clip(
+        RoundedCornerShape(10.dp))) {
+        ListItem.Item(
+            text = "Text",
+            onClick = {},
+            textAlignment = ListItemTextAlignment.Regular,
+            border = BorderType.Bottom,
+            trailingAccessoryContent = { Icon(icon = FluentIcon(Icons.Outlined.KeyboardArrowRight))},
+        )
+        ListItem.Item(
+            text = "Text",
+            onClick = {},
+            textAlignment = ListItemTextAlignment.Regular,
+            border = BorderType.TopBottom,
+            trailingAccessoryContent = { Icon(icon = FluentIcon(Icons.Outlined.KeyboardArrowRight))},
+        )
+        ListItem.Item(
+            text = "Text",
+            onClick = {},
+            textAlignment = ListItemTextAlignment.Regular,
+            border = BorderType.TopBottom,
+            trailingAccessoryContent = { Icon(icon = FluentIcon(Icons.Outlined.KeyboardArrowRight))},
+        )
+        ListItem.Item(
+            text = "Text",
+            onClick = {},
+            textAlignment = ListItemTextAlignment.Regular,
+            border = BorderType.Top,
+            trailingAccessoryContent = { Icon(icon = FluentIcon(Icons.Outlined.KeyboardArrowRight))},
+        )
+    }
+    ListItem.SectionDescription(description = "Grouped list containing multiple similar elements", modifier = Modifier.wrapContentHeight().padding(0.dp))
+}
+
```

Comment:
please add both before and after screenshots

Reply:
1. This did not exist before, so only added the after screenshot. 
2. The requirement request for a group list Item as per a provided Figma. I added it to the Demo App to dismiss the requirement for a separate component and show that ListItem covers the required functionality already.